subsys/bluetooth/audio/has_internal.h

```diff
@@ -8,9 +8,121 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#define BT_HAS_FEAT_HEARING_AID_TYPE_MASK 0x03
+/* Control Point opcodes */
+#define BT_HAS_OP_READ_PRESET_REQ        0x01
+#define BT_HAS_OP_READ_PRESET_RSP        0x02
+#define BT_HAS_OP_PRESET_CHANGED         0x03
+#define BT_HAS_OP_WRITE_PRESET_NAME      0x04
+#define BT_HAS_OP_SET_ACTIVE_PRESET      0x05
+#define BT_HAS_OP_SET_NEXT_PRESET        0x06
+#define BT_HAS_OP_SET_PREV_PRESET        0x07
+#define BT_HAS_OP_SET_ACTIVE_PRESET_SYNC 0x08
+#define BT_HAS_OP_SET_NEXT_PRESET_SYNC   0x09
+#define BT_HAS_OP_SET_PREV_PRESET_SYNC   0x0a
+
+/* Application error codes */
+#define BT_HAS_ERR_INVALID_OPCODE         0x80
+#define BT_HAS_ERR_WRITE_NAME_NOT_ALLOWED 0x81
+#define BT_HAS_ERR_PRESET_SYNC_NOT_SUPP   0x82
+#define BT_HAS_ERR_OPERATION_NOT_POSSIBLE 0x83
+#define BT_HAS_ERR_INVALID_PARAM_LEN      0x84
+
+/* Hearing Aid Feature bits */
+#define BT_HAS_FEAT_HEARING_AID_TYPE_LSB  0
+#define BT_HAS_FEAT_HEARING_AID_TYPE_MSB  1
+#define BT_HAS_FEAT_PRESET_SYNC_SUPP      2
+#define BT_HAS_FEAT_INDEPENDENT_PRESETS   3
+#define BT_HAS_FEAT_DYNAMIC_PRESETS       4
+#define BT_HAS_FEAT_WRITABLE_PRESETS_SUPP 5
```

Comment:
Do you ever need to use these values without first passing them through the `BIT()` macro? If not, I'd suggest defining them as the actual bitmasks directly rather than requiring always explicit use of `BIT()`.

Reply:
done